src/test/java/games/strategy/engine/config/client/remote/LobbyServerPropertiesFetcherTest.java

```diff
@@ -17,7 +17,7 @@
 import games.strategy.engine.lobby.client.login.LobbyServerProperties;
 import games.strategy.util.Version;
 
-@RunWith(MockitoJUnitRunner.class)
+@RunWith(MockitoJUnitRunner.StrictStubs.class)
```

Comment:
`StrictStubs` will become the default runner in Mockito 3 (just as `Strict` is the default runner in Mockito 2).  We should probably add a backlog item to remind us to revert all these changes back to just `MockitoJUnitRunner.class` when we upgrade to Mockito 3.  @RoiEXLab, can you please add that to the backlog once this PR is merged?